src/Nancy/GlobalizationConfiguration.cs

```diff
@@ -15,15 +15,22 @@ public class GlobalizationConfiguration
         /// <summary>
         /// Initializes a new instance of the <see cref="GlobalizationConfiguration"/> class
         /// </summary>
-        /// <param name="supportedCultureNames"></param>
-        public GlobalizationConfiguration(IEnumerable<string> supportedCultureNames)
+        /// <param name="supportedCultureNames">An array of supported cultures</param>
+        /// <param name="defaultCulture">The default culture of the application</param>

```

Comment:
I think we need to point out that the culture you provide here, needs to exist in the `supportedCultureNames` collection and handle it appropriately when it does not. Maybe we need to introduce a general purpose `ConfigurationException` for these types of things?